tools/Readme.txt

```diff
@@ -0,0 +1,64 @@
+This tool aims to cleanup MAFs in the following aspects:
+
+1. Convert dates back to gene symbols (hugo_symbol)
+
+SEPT1	1-Sep
```

Comment:
this list covers all the cases we observed so far .. but maybe we should have a reverse mapping for all dates back to what the gene that created the date would be. .. so if we encounter a new "date" conversion case in the future we will be able to fix it without needing to modify this script.